beta/nncf/tensorflow/quantization/quantizers.py

```diff
@@ -264,16 +271,43 @@ def build(self, input_shape, input_type, name, layer):
             'signed_var': signed
         }
 
-    def quantize(self, inputs, weights):
-        return symmetric_quantize(
-            inputs,
-            weights['scale_var'],
-            weights['signed_var'],
-            num_bits=self.num_bits,
-            per_channel=self.per_channel,
-            narrow_range=self.narrow_range,
-            eps=self._eps
-        )
+    def apply_saturation_fix(self, weights):
+        if self.num_bits != 8 or not self._half_range:
+            raise RuntimeError('Attempt to apply saturation issue fix '
+                               'to quantizer which is not configured for that.')
+
+        multiplier = 127./63. if self.narrow_range else 255./127.
```

Comment:
Could you please add a short comment about what these "magic numbers" mean?

```suggestion
        multiplier = 127 / 63 if self.narrow_range else 255 / 127
```